DatadogWebViewTracking/Sources/WebViewTrackingMessage.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+public typealias JSON = [String: Any]
+
+/// Intermediate type to parse WebView messages and send them to the message bus
+public enum WebViewTrackingMessage {
+    /// A log message with a JSON payload
+    case log(JSON)
+
+    /// A RUM event with a JSON payload
+    case rumEvent(JSON)
```

Comment:
/nit I like this enum. About the case names, I think it should be `.log` and `.rum` **or** `.logEvent` and `.rumEvent`

Reply:
went with no Event suffix.